db/openshift.deploy.yml

```diff
@@ -25,7 +25,11 @@ parameters:
     required: true
     value: 1Gi
   - name: REGISTRY
+    description: Container registry to import from (internal is image-registry.openshift-image-registry.svc:5000)
```

Comment:
Don't understand this comment if we are no longer using imagestream

Reply:
You can still use imagestreams or the internal oc registry.  The important part of the comment is outside the parens.  We can remove if that's misleading or just useless.